.cirrus.yml

```diff
@@ -957,6 +1083,41 @@ artifacts_task:
           type: application/octet-stream
 
 
+artifacts_aarch64_task:
+    name: "Artifacts (aarch64)"
+    alias: artifacts_aarch64
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: *not_docs_multiarch
+    depends_on:
+        - success
+    ec2_instance: *standard_build_ec2_aarch64
+    # This task is a secondary/convenience for downstream consumers, don't
+    # block development progress if there is a failure in a PR, only break
+    # when running on branches or tags.
+    allow_failures: $CIRRUS_PR != ''
+    container: *smallcontainer
+    env:
+        CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
+        TEST_ENVIRON: container
+    # In order to keep the download URL and Cirrus-CI artifact.zip contents
+    # simple, nothing should exist in $CIRRUS_WORKING_DIR except for artifacts.
+    clone_script: *noop
+    fedora_binaries_script:
+        - mkdir -p /tmp/fed
+        - cd /tmp/fed
+        - $ARTCURL/build_aarch64/repo/repo.tbz
+        - tar xjf repo.tbz
+        - cp ./bin/* $CIRRUS_WORKING_DIR/
+    always:
+      contents_script: ls -la $CIRRUS_WORKING_DIR
+      # Produce downloadable files and an automatic zip-file accessible
+      # by a consistent URL, based on contents of $CIRRUS_WORKING_DIR
+      # Ref: https://cirrus-ci.org/guide/writing-tasks/#latest-build-artifacts
+      binary_artifacts:
+          path: ./*
+          type: application/octet-stream
+
+
```

Comment:
I'd prefer you just leave out the aarch64 artifacts task.  Let's wait until there's an actual need.  At that time, it should be included into the single common artifacts task (i.e. not a separate one) - to make downstream integration easier.

Reply:
removed artifacts_aarch64.